tests/tests/test_integration/test_registration.py

```diff
@@ -1,22 +1,20 @@
 import os
 import platform
 import sys
+from pathlib import Path
 
 import numpy as np
 import pandas as pd
+import pytest
 from imio.load import load_any
 
 from brainreg.cli import main as brainreg_run
 
-data_dir = os.path.join(
-    os.getcwd(),
-    "tests",
-    "data",
-    "brain data",
-)
+test_data_dir = Path(os.getcwd()) / "tests" / "data"
 
-test_niftyreg_output = os.path.join(
-    os.getcwd(), "tests", "data", "registration_output", platform.system()
+brain_data_dir = test_data_dir / "brain data"
+expected_niftyreg_output_dir = (
+    test_data_dir / "registration_output" / platform.system()
```

Comment:
Just curious to know whether there would be a difference with chip architectures? `platform.system()` evaluates to `Darwin` on macOS